src/sass/_animations.scss

```diff
@@ -0,0 +1,35 @@
+.pulse {
```

Comment:
Another note: the naming convention of using the underscore before certain sass files just implies that this is a partial file, and will only be converted and used if imported in a full sass file. This is just a sass convention